tee-worker/enclave-runtime/src/top_pool_execution.rs

```diff
@@ -185,10 +185,13 @@ fn execute_top_pool_trusted_calls_internal() -> Result<()> {
 		&mut LastSlot,
 	)? {
 		Some(slot) => {
-			if slot.duration_remaining().is_none() {
-				warn!("No time remaining in slot, skipping AURA execution");
-				return Ok(())
-			}
+			// FIXME: To disable this checking is a temp solution for the moment.
+			// Later we need to investigate and double check again whether this is
+			// a good solution.
+			// if slot.duration_remaining().is_none() {
```

Comment:
Can we keep this check and run it only if there are more than 1 enclave registered ?

Reply:
"Later we need to investigate" => I doubt if we'll do that...